File: RiskAssessment/API/RiskManagement.Web/src/app/pages/login/login.model.ts
export class ResetPassswordModel {
    public Password: string;
    public ConfirmPassword: string;
    public SecurityQuestion1: number;
    public SecurityAnswer1: string;
    public SecurityQuestion2: number;
    public SecurityAnswer2: string;
    constructor() {
    }
}

export class SecurityQustionOfUserModel {
    public SecurityQuestion1: string;
    public SecurityQuestion2: string;
}

export class LggedInUserModel {
    public UserId: number = 0;
    public Message: string;
    public RoleId: number;
    public Token: string;
    public IsNewUser: boolean = false;
    public UserName: string;
    public Password: string;
    public RoleList: number[] = [];
}

File: Angular2_AspDotNet/API/Angular2_AspDotNet.Models/LoginModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Angular2_AspDotNet.Models
{
    public class LoginModel
    {
        public LoginModel()
        {
            this.model = new Dictionary<string, SessionData>();
        }
        public Dictionary<string, SessionData> model { get; set; }
    }
    public class SessionData
    {
        public SessionData()
        {
        }
        public int UserId { get; set; }
        public string Token { get; set; }
        public DateTime ExpirationDate { get; set; }
        public int OrganizationId { get; set; }
        public List<int> RoleId { get; set; } = new List<int>();
    }

    public class LoggedInUserModel
    {
        public int UserId { get; set; } = 0;
        public string Message { get; set; }
        public Nullable<int> RoleId { get; set; }
        public string Token { get; set; }
        public bool IsNewUser { get; set; } = false;

        public string UserName { get; set; }
        public string Password { get; set; }
        public int OrganizationId { get; set; }
        public List<int> RoleList { get; set; } = new List<int>();

    }
}
